yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -95,6 +95,15 @@ func (d *TableIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 				))
 			}
 
+			if c.ConstraintType == queryparser.FOREIGN_CONSTR_TYPE && d.partitionTablesMap[c.ReferencedTable] {
+				issues = append(issues, NewForeignKeyReferencesPartitionedTableIssue(
+					TABLE_OBJECT_TYPE,
+					table.GetObjectName(),
+					"",
```

Comment:
lets add sqlstatement also here?
Moving forward(flattening of issues in assessment report), we will have this field for each issue and its good to have sql statement available in each case.

Reply:
SQl statement is already added for these later in the GetDDLIssues

Reply:
hmm.. not sure what is the reason of adding it there.
But adding at the time of issue creation makes more sense to me.
is the sql statement getting modified somewhere in between.

Reply:
No not changing sql statement anywhere, it's just that skipping the overhead of passing the query to these `ddl_detectors`, so adding it in one go there itself.